### DISCUSSION TITLE
Prowler APP Automated scans

### DISCUSSION THREAD

**Discussion by vinayhr5** [prowler-cloud/prowler#6235] [Q&A] [answered]
Hi Prowler Team,

Is there any way in Prowler APP, to schedule a scan on a particular time. for eg. 12 hrs, 8hrs etc.
I am aware that Prowler will automatically scan all configured providers every 24 hours. 

Do we have any api ? or somthing else which prowler provides?

**Comment by jfagoagas** [comment, marked as answer]
Hello @vinayhr5 that is present in our current roadmap, you can check it at https://roadmap.prowler.com. Until that you can use the Scan API https://api.prowler.com/api/v1/docs#tag/Scan/operation/scans_create to launch scans manually.

**Reply by vinayhr5** [reply]
Thanks
